requirements.txt

```
@@ -7,6 +7,6 @@ djangorestframework==3.14.0
 types-pytz==2022.7.1.2
 types-requests==2.28.11.15
 types-urllib3==1.26.25.8
-django-stubs==1.15.0
+git+https://github.com/typeddjango/django-stubs@update-mypy
```

TODO revert when django-stubs is released before merging this!

```suggestion
django-stubs==1.16.0
```